### ISSUE TITLE
Run npm install geo-tz on Windows machine. When running got missing dependencies

### ISSUE THREAD

**Issue by vkelman** [evansiroky/node-geo-tz#19]
Run npm install geo-tz on Windows machine. When running my app got missing shpjs dependency.
Run npm install shpjs. When running my app got missing turf-featurecollection dependency.

I noticed that in geo-tz package.json both shpjs and turf-featurecollection are listed in devDependencies section. Should they be moved to dependencies section?  Any other dependencies?

  Thank you

**Comment by evansiroky** [comment]
What version of node.js and npm are you using?

**Comment by vkelman** [comment]
It's node.js version 6.5.0.
npm -v npm gives 3.10.7

**Comment by vkelman** [comment]
I added the following to package.json of **my** application:

```
    "shpjs": "^3.3.2",
    "turf-featurecollection": "^1.0.1",
    "jsts": "^1.1.2",
    "turf-polygon": "^1.0.3",

```
 geo-tz began to work.
I

**Comment by evansiroky** [comment]
Those dev dependencies are solely for updating the timezone data, but it looks like I am requiring them anyways from the root module.  I'll get this fixed after I resolve #14.

**Comment by vkelman** [comment]
Thank you, Evan.  In the past I wrote some C# library using Noda Time, downloaded and exported into SQL Server shapefiles - all that to deal with timezones in our C# application. Was glad to find ready-to-use module for our Node.js.

**Comment by vkelman** [comment]
Closed by mistake, the reopened.

**Comment by evansiroky** [comment]
Fixed with #23
